ridlbe/c++11/templates/cli/src/union_anyop.erb

```diff
@@ -11,14 +11,11 @@ void operator<<= (TAOX11_CORBA::Any &any, <%= scoped_cxx_in_type %> _union )
 % else
       TAOX11_NAMESPACE::CDR_Marshal_Policy
 % end
-        >::insert_copy (
-          any,
-          <%= scoped_cxx_typecode %>,
-          _union);
+        >::insert_copy (any, <%= scoped_cxx_typecode %>, _union);
```

Comment:
Why do you feel this kind of change is an improvement?

Reply:
Was looking through the generated code and with anytypecode enabled this adds a lot of sloc without a real value, so just passed all arguments in one line

Reply:
Yes, but <%= scoped_cxx_typecode %> can expand to quite a loooong name making a single line a lot less readable. Why this obsession with sloc?

Reply:
Much wider screen then high ;-)